system/default_ad_api_helpers/src/macros/create_node.hpp

```diff
@@ -0,0 +1,37 @@
+// Copyright 2022 Autoware Foundation
```

Comment:
Where is this file from? If you use the AWF's license notation, please link the original file to track it.

Reply:
Done https://github.com/autowarefoundation/autoware.universe/pull/1494/commits/7eda408a79b0ed4e8f69d987c8cf413deeabd93c.